plugins/regmap/src/register/registersimplewidget.cpp

```diff
@@ -135,20 +150,20 @@ void RegisterSimpleWidget::checkPreferences()
 	if(background.contains("Register background")) {
 		bool ok;
 		regBaseInfoWidget->setStyleSheet(
-			QString("background-color: " + Util::getColors().at(value->text().toInt(&ok, 16) % 16)));
+			QString("background-color: " + Util::getColors().at(value->text().toUInt(&ok, 16) % 16)));
 	}
 
 	if(background.contains("Register text")) {
 		bool ok;
-		value->setStyleSheet(QString("color: " + Util::getColors().at(value->text().toInt(&ok, 16) % 16)));
+		value->setStyleSheet(QString("color: " + Util::getColors().at(value->text().toUInt(&ok, 16) % 16)));
```

Comment:
This line sets the stylesheet, gets colors, searches through a color map, converts from text to uint, and then uses modulo to wrap around  the color map ..